pkgs/tools/system/testdisk/default.nix

```diff
@@ -0,0 +1,71 @@
+{ mkDerivation
+, stdenv
+, fetchurl
+, ncurses
+, libuuid
+, pkgconfig
+, enableJpeg ? false, libjpeg ? null
```

Comment:
Do we need so many options? Cannot users who want to disable `libjpeg` just do `testdisk.override { libjpeg = null; }`?

Reply:
Yeah, they could. For some reason I was on some explicit train at that point. I will flip at least libjpeg.